lib/resources/r4/appointment_patch.yaml

```diff
@@ -63,6 +63,32 @@ operations:
         <li>CodeableConcept.text is required and CodeableConcept.coding, if present, will be ignored</li>
         <li>When the value array is empty, reasonCode will be removed</li>
       </ul>
+  - name: add-cancelationReason
+    path: /cancelationReason
+    operation: add
+    type: CodeableConcept
+    url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.cancelationReason
+    description: The coded reason for the appointment being cancelled.
```

Comment:
The other descriptions don't have a period on the description. Maybe we should remove this one too?

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/593/commits/75d6c9ecb797dd7b1b65802f8ab08988b208802a#diff-1f6c4e0ab0717eb6bc3a7078d167381eab7bcfa504e624c91697dccbad1705ffL71